src/randoop/reflection/ReflectionManager.java

```diff
@@ -107,8 +107,28 @@ public void apply(Class<?> c) {
     
   }
 
-
-
+  /**
+   * Applies the visitors to the methods of the class. 
+   * Will visit all methods unless visiting only public members.
+   * 
+   * @param c  the class whose methods should be visited
+   */
+  private void applyMethods(Class<?> c) {
+    Set<Method> methods = new HashSet<>();
```

Comment:
The point of this set methods is to avoid visiting a method twice. The fact that it wasn't doing its job was hidden by the facts that visiting twice has no side-effect other than wasting time, and that the collected objects are placed in a set by the OperationExtractor visitor.